### PR TITLE
wcag fixes

### PR DESCRIPTION
**Pull request by maxvisser**
teamforus/forus#2222
teamforus/forus#2219

### CODE REVIEW

**Comment by maxvisser** [comment]
@SjoerdHilhorst could you help with listing the issues that you solved? (from forus)

File: src/sources/forus-webshop/pug/tpl/pages/provider.pug [hunk 1]
```diff
@@ -18,7 +18,7 @@ block content
                 .pane-head Aanbieder informatie
                 .pane-section: .provider-details
                     .provider-description
-                        img(ng-src="{{ $ctrl.provider.logo.sizes.thumbnail || '/assets/img/placeholders/organization-thumbnail.png' }}").provider-logo
+                        img(ng-src="{{ $ctrl.provider.logo.sizes.thumbnail || '/assets/img/placeholders/organization-thumbnail.png' }}" alt="Aanbieder logo").provider-logo
```
Comment (maxvisser) [inline src/sources/forus-webshop/pug/tpl/pages/provider.pug]: As it is aswell sometimes a thumbnail the alt text is not correct. I believe this image is purely decorative; maybe just add the alt text and leave it empty

File: src/sources/forus-webshop/pug/tpl/directives/app-footer.pug [hunk 2]
```diff
@@ -16,9 +16,9 @@ section.section.section-footer
                 .block.block-markdown
                     h3 Download Me-app
                     .block.block-app_download
-                        a(href="{{ $root.appConfigs.android_link }}" target="_blank" aria-label="Download Me-app")
+                        a(href="{{ $root.appConfigs.android_link }}" target="_blank")
```
Comment (maxvisser) [inline src/sources/forus-webshop/pug/tpl/directives/app-footer.pug:19]: maybe we should add here role="button"

Comment (maxvisser) [reply src/sources/forus-webshop/pug/tpl/directives/app-footer.pug:19]: should check in the document

File: src/sources/forus-webshop/pug/tpl/directives/app-footer.pug [hunk 3]
```diff
@@ -16,9 +16,9 @@ section.section.section-footer
...
                             img(src="./assets/img/icon-app/icon-app-android.svg" alt="Google Play Store")
-                        a(href="{{ $root.appConfigs.ios_iphone_link }}" target="_blank" aria-label="Download Me-app")
+                        a(href="{{ $root.appConfigs.ios_iphone_link }}" target="_blank")
                             img(src="./assets/img/icon-app/app-store-badge.svg" alt="Apple App Store")
```
Comment (maxvisser) [inline src/sources/forus-webshop/pug/tpl/directives/app-footer.pug]: Alt text should contain whats visible on the image

File: src/sources/forus-webshop/pug/tpl/directives/app-footer.pug [hunk 4]
```diff
@@ -16,9 +16,9 @@ section.section.section-footer
                 .block.block-markdown
                     h3 Download Me-app
                     .block.block-app_download
-                        a(href="{{ $root.appConfigs.android_link }}" target="_blank" aria-label="Download Me-app")
+                        a(href="{{ $root.appConfigs.android_link }}" target="_blank")
                             img(src="./assets/img/icon-app/icon-app-android.svg" alt="Google Play Store")
```
Comment (maxvisser) [inline src/sources/forus-webshop/pug/tpl/directives/app-footer.pug]: Alt text should contain whats visible on the image

**Comment by maxvisser** [comment]
teamforus/forus#2222

**Comment by SjoerdHilhorst** [comment]
all issues that I've solved are under the QA column in Zenhub

**Pull request merged**